auth-web/src/util/constants.ts

```diff
@@ -247,6 +247,87 @@ export enum NrEntityType {
+  CTC = 'Continuation In', // Cooperative
+  ULCT = 'Continuation In', // Unlimited Liability Co.
+
+  // Conversion
+  BECR = 'Conversion (Act)', // BC Benefit Company Incorporation
+  CCV = 'Conversion (Act)', // Community Contribution Co.
+  UC = 'Conversion (Act)', // Unlimited Liability Co.
+
+  // Incorporation
+  BC = 'Incorporation', // BC Benefit Company Incorporation
+  CC = 'Incorporation', // Community Contribution Co.
+  CP = 'Incorporation', // Cooperative
+  CR = 'Incorporation', // B.C. Company
+  FI = 'Incorporation', // Financial Institution (BC)
+  PA = 'Incorporation', // Private Act
+  PAR = 'Incorporation', // Parish
+  UL = 'Incorporation', // Unlimited Liability Co.
+  XCR = 'Incorporation', // Corporation (Foreign)
+  XCP = 'Incorporation', // Extraprovincial Cooperative
+  XUL = 'Incorporation', // Extraprovincial Unlimited Liability Company
+
+  // Name Change
+  BEC = 'Name Change', // BC Benefit Company Incorporation
+  CCC = 'Name Change', // Community Contribution Co.
+  CCP = 'Name Change', // Cooperative
+  CCR = 'Name Change', // B.C. Company
+  CFI = 'Name Change', // Financial Institution (BC)
+  CFR = 'Name Change',
+  CFR_DBA = 'Name Change', // Sole Proprietorship (DBA)
+  CFR_FR = 'Name Change', // Sole Proprietorship
+  CFR_GP = 'Name Change', // General Partnership
+  CLC = 'Name Change', // Extraprovincial Limited Liability Company
+  CLL = 'Name Change', // Limited Liability Partnership
+  CLP = 'Name Change', // Limited Partnership
+  CUL = 'Name Change', // Unlimited Liability Co.
+  XCCP = 'Name Change', // Extraprovincial Cooperative
+  XCCR = 'Name Change', // Corporation (Foreign)
+  XCLL = 'Name Change', // Extraprovincial Limited Liability Partnership
+  XCLP = 'Name Change', // Extraprovincial Limited Partnership
+
+  // Registration
+  FR = 'Registration', // Sole Proprietorship/General Partnership/DBA - Registration
+  FR_DBA = 'Registration', // Sole Proprietorship (DBA)
+  FR_FR = 'Registration', // Sole Proprietorship
+  FR_GP = 'Registration', // General Partnership
+  LC = 'Registration', // Extraprovincial Limited Liability Company
+  LL = 'Registration', // Limited Liability Partnership
+  LP = 'Registration', // Limited Partnership
+  XLL = 'Registration', // Extraprovincial Limited Liability Partnership
+  XLP = 'Registration', // Extraprovincial Limited Partnership
+
+  // Reinstatement
+  RLC = 'Reinstatement', // Extraprovincial Limited Liability Company
+  XRCR = 'Reinstatement', // Corporation (Foreign)
+
+  // Restoration
+  BERE = 'Restoration', // BC Benefit Company Incorporation
+  RCC = 'Restoration', // Community Contribution Co.
+  RCP = 'Restoration', // Cooperative
+  RCR = 'Restoration', // B.C. Company
+  RFI = 'Restoration', // Financial Institution (BC)
+  RSO = 'Restoration',
+  RUL = 'Restoration', // Unlimited Liability Co.
+  XRCP = 'Restoration', // Extraprovincial Cooperative
+  XRUL = 'Restoration' // Extraprovincial Unlimited Liability Company
+}
```

Comment:
Looks GGRREEAATT! Do you think you should pass this by Mihai or are you pretty confident in it?

Reply:
I'm fairly confident but I will check with Mihai if he wants to have a look over it